examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<ul><li>"virtually flat" could be simplified as "flat" or "has almost zero slope"?</li><li><em>the per-class histograms are simply overlapping too much to offer much discriminative power to the linear model -&gt; the overlap in the per-class histograms of this feature show that it does not increase the model discriminative power</em></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>